test/test_helper.rb

```diff
@@ -6,10 +6,14 @@
 require 'single_cov'
 SingleCov.setup :minitest
 
+require 'rails'
 require 'maxitest/global_must'
 require 'maxitest/autorun'
-require 'mocha/setup'
-require 'rails'
+if Rails::VERSION::MAJOR >= 7
+  require 'mocha/minitest'
```

Comment:
this should already work on older versions

Reply:
... maybe just keep it for now, otherwise we have to update the old lockfiles and that might cause even more chaos